### PR TITLE
[Merged by Bors] - fix multiarch docker builds

### PR DESCRIPTION
**Pull request by antondlr**
## Issue Addressed

#3902 
Tested and confirmed working [here](https://github.com/antondlr/lighthouse/actions/runs/3970418322)

## Additional Info

buildx v0.10.0 added provenance attestations to images but they are packed in a way that's incompatible with `docker manifest`
https://github.com/docker/buildx/releases

### CODE REVIEW

**Review by realbigsean — APPROVED** [review]
Nice! Thanks anton!

**Comment by michaelsproul** [comment]
Awesome, thanks Anton!

I'll back-merge this to Capella once it lands in unstable

**Pull request closed without merging**
